docs/cloud/2-output-streams/5-gcp-pubsub/1-overview.md

```diff
@@ -0,0 +1,123 @@
+---
+id: overview
+title: Overview
+slug: /cloud/output-streams/gcp-pubsub
+---
+
+## GCP PuSub
+
+GCP PubSub is a fully managed, real-time data ingestion service that can be used to consume Golioth events in a simple, trusted, and scalable way. You can ingest data in multiple ways inside GCP with support for popular protocols, including AMQP, HTTPS, and Apache Kafka. You can also consume GCP PubSub in a serverless manner using GCP Cloud Functions.
+
+### GCP PubSub Specific Attributes
+
+For each Output Stream type, there is a set of specify attributes. Here are the ones for GCP PubSub:
+
+| Attribute        | Type   | Required | Description                                                                                                                                                                                    |
+| ---------------- | ------ | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| serviceAccount | json | ✅       | Full JSON object representing the gcp service account to reach GCP PubSub. |
```

Comment:
using `gcp` as all lower case vs all upper case. I think it should be all upper case.